shared/src/main/scala/amf/client/remod/AMFRender.scala

```diff
@@ -0,0 +1,12 @@
+package amf.client.remod
+
+import amf.core.model.document.BaseUnit
+import org.yaml.model.YDocument
+
+private[remod] object AMFRender {
```

Comment:
Why have separate methods instead of writers or something like that? Same signature but writing is changed. Both writers would receive YNodes